api/orgs.js

```diff
@@ -21,6 +21,9 @@ const QUERY_DICTIONARY = new Map([
 	["edition", "o.edition"],
 	["instance", "o.instance"],
 	["type", "o.type"],
+	["location", "o.org_location"],
+	["env", "o.org_env"],
+	["release", "o.org_release"],
```

Comment:
org_xxx is not I think the right name here.  The table is org.  org.org_xxx is redundant.  org_location can probably be location, org_release can be release or release_schedule.  org_env can just be environment.